### ISSUE TITLE
Feature request: `recreate` boolean option to Indicate if previous comment should be removed before creating a new comment

### ISSUE THREAD

**Issue by blordpluto** [ArtiomTr/jest-coverage-report-action#288]
### Idea

On long-lived PRs, there is the potential for the number of comments to become spammy and repetitive. Typically, a reviewer is only interested in the most recent run of the latest HEAD, and not the history of the branch. Thus, it would be useful to delete existing coverage comments from a PR before adding a new coverage comment.

Some other actions have dealt with this through a `recreate` option, e.g. https://github.com/marketplace/actions/sticky-pull-request-comment#recreate

Please let me know if this is something you would be interested in receiving a PR for.

**Comment by ArtiomTr** [comment]
Hello @blordpluto :wave:,

This action creates a report comment only on the first run, on further runs, it just edits the first comment. If you see that action repeatedly generates new comments on each run, that is a bug and should be fixed. If you propose an ability to change standard behaviour, and introduce a new "recreation" mechanism, that will delete the previous comment and create a new one, I would like to make sure that this feature will bring more benefits, than complexity to this action. In my opinion, the "recreation" mechanism could be helpful to stick the comment to the bottom of a pull request. On the other hand, I think it is better to keep as few options as possible to avoid confusing users. Maybe, the default behaviour should mix these two approaches. For instance, the algorithm could look like this:
1. If the amount of commits/comments/other actions on PR is relatively small, edit the previous comment.
2. Else, recreate the comment to "push" it to the bottom of PR.

So, to sum up, I don't know the proper solution for this problem, as I have never tested this action on large PR's. I would like to hear different opinions on this issue. Also, if it is not difficult for you, could you please send a screenshot or a link to a large PR that uses this action? Just want to see how it looks and understand the problem more clearly.

**Comment by ArtiomTr** [comment]
In case anyone wants the same behaviour, now action supports the generation of custom output. This means you have full control on how comments are created, you can use [sticky-pull-request-comment](https://github.com/marocchino/sticky-pull-request-comment) action. See [documentation about `output` option](https://github.com/ArtiomTr/jest-coverage-report-action#outputs).
